prow/kube/cluster.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kube
```

Comment:
so there's already https://github.com/kubernetes/test-infra/blob/master/prow/flagutil/kubernetes.go, maybe we should merge some logic, or it's time to unify all the different kubeclients we have? @stevekuznetsov

Reply:
that's the horrible client that is not compatible with kubeconfig. I think we should kill this.

Reply:
yeah, I mean, let everything use this :-)